docs/ARCHITECTURE.md

```diff
@@ -201,3 +201,67 @@ can be found under various directories such as `factory/`, `modprobe.d/`, `netwo
 `tools/`, `coccinelle/`, `.github/`, `.semaphore/`, `.mkosi/` host various
 utilities and scripts that are used by maintainers and developers. They are not
 shipped or installed.
+
```

Comment:
is ARCHITECTURE.md really the right place for this? so far this was mostly a document for peope wanting to hack on systemd.

maybe move this to SECURITY.md (and change the title of that doc a bit to be a bit more generic)

or make this a doc of its own?

Reply:
I'll add a new readme in src/boot - I think I want to avoid this showing up on systemd.io after all, as it's not really something targeted to end users